platform/src/main/java/org/stellar/anchor/platform/service/PaymentOperationToEventListener.java

```diff
@@ -66,30 +73,55 @@ public void onSent(PaymentOperationResponse payment) {
   }
 
   private void sendToQueue(TransactionEvent event) {
-    // TODO: Send the event to event API.
+    eventService.publish(event);
     System.out.println("Sent to event queue" + new Gson().toJson(event));
   }
 
   TransactionEvent receivedPaymentToEvent(Sep31Transaction txn, PaymentOperationResponse payment) {
-    TransactionEvent txnEvent =
-        TransactionEvent.builder()
-            .transactionId(txn.getId())
-            .status(txn.getStatus())
-            .amountIn(new Amount(txn.getAmountIn(), txn.getAmountInAsset()))
-            .amountOut(new Amount(txn.getAmountOut(), txn.getAmountOutAsset()))
-            .amountFee(new Amount(txn.getAmountFee(), txn.getAmountFeeAsset()))
-            .stellarTransactions(
-                StellarTransaction.builder()
-                    .id(txn.getStellarTransactionId())
-                    .memo(txn.getStellarMemo())
-                    .memoType(txn.getStellarMemoType())
-
-                    // TODO: payment does not provide access to createdAt timestamp. Need to submit
-                    // a PR.
-                    .build())
-            .build();
+    TransactionEvent event = TransactionEvent.builder()
+      .eventId(UUID.randomUUID().toString())
+      .type(org.stellar.anchor.event.models.TransactionEvent.Type.TRANSACTION_CREATED)
+      .id(txn.getId())
+      .status(TransactionEvent.Status.valueOf(txn.getStatus()))
+      .sep(org.stellar.anchor.event.models.TransactionEvent.Sep.SEP_31)
+      .kind(org.stellar.anchor.event.models.TransactionEvent.Kind.RECEIVE)
+      .amountIn(org.stellar.anchor.event.models.Amount.builder()
+              .amount(txn.getAmountIn())
+              .asset(txn.getAmountInAsset())
+              .build()
+      )
+      .amountOut(org.stellar.anchor.event.models.Amount.builder()
```

Comment:
Do you mind removing the `org.stellar.anchor.event.models..*` full paths and importing the classes on the top of the file instead?